### ISSUE TITLE
Changing brightness fails

### ISSUE THREAD

**Issue by charlesfracchia** [nmelo/homebridge-lifx#12]
Currently, setting the brightness in the Home app using `homebridge-lifx` fails.
Tracked it down to [this line](https://github.com/nmelo/homebridge-lifx/blob/825e7a46be202ff73a3331c7f115ab1db69d7b30/index.js#L210), which is setting `null` for the [`power_on` parameter of the LIFX api `setColor` function](https://github.com/ywadi/lifx#function-setcolorselector-color-duration-power_on-cb)

**Comment by Kazom** [comment]
@charlesfracchia I don't think this plugin is still being actively developed especially after lifx added native support. 
Theres a slightly newer one that also seems dead but has always worked well for my non natively supported bulbs.
https://github.com/devbobo/homebridge-lifx-lan
